### ISSUE TITLE
Bug in JumpStatementsSpacing

### ISSUE THREAD

**Issue by dereuromark** [slevomat/coding-standard#903]
```php
            if ($repository['private']) { // Some comment
                continue;
            }
```

resulted in unexpected try to fix (which even failed)

    QueueRepoSyncTask.php  FAILED TO FIX

Running check only reveals the issue:
```
 76 | ERROR | [x] Expected 1 lines before "continue", found 0.
    |       |     (SlevomatCodingStandard.ControlStructures.JumpStatementsSpacing.IncorrectLinesCountBeforeControlStructure)
```
Seems like the comment in that line wrongly triggers the sniff.
It should either ignore comments or detect that those are in the same line as the code before and thus not relevant to create newlines.

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/261d023d288ed8a31b07e2aa554ff4b0125c929d
